src/common/snap_types.h

```diff
@@ -52,15 +52,17 @@ struct SnapRealmInfoNew {
   SnapRealmInfo info;
   utime_t last_modified;
   uint64_t change_attr;
+  bool is_snapdir_visible;
```

Comment:
`uint64_t flags` for future additions.

`bool` is a waste of payload space.

Reply:
PTAL https://github.com/ceph/ceph/pull/64345/commits/d3c2d4d278b6a50e2569b19d9370ae60d1729e6a